sbirez/static/js/controllers/proposalReport.js

```diff
@@ -9,25 +9,33 @@ angular.module('sbirezApp')
     $scope.goodStartWorkflow = null;
     $scope.buttonMessage = 'Get Started';
     $rootScope.bodyClass = 'proposal proposal-overview';
+    $scope.showContinue = true;
+    var first = true;
 
     var goodStartElement = function() {
-      var goodStart = $scope.overview[0].id;
+      var goodStart = -1; //$scope.overview[0].id;
```

Comment:
I don't actually understand this logic, just want to make sure this comment-out doesn't represent a temporary then forgotten hack